tools/target-browser/backup.go

```diff
@@ -26,14 +26,15 @@ type BackupListOptions struct {
 
 // Backup struct stores extracted fields from actual Backup API GET response
 type Backup struct {
-	Name           string `json:"Name"`
-	UID            string `json:"UID"`
-	Type           string `json:"Type"`
-	Size           string `json:"Size"`
-	Status         string `json:"Status"`
-	BackupPlanUID  string `json:"BackupPlan UID"`
-	CreationTime   string `json:"Start Time"`
-	CompletionTime string `json:"End Time"`
+	Name                 string `json:"Name"`
+	UID                  string `json:"UID"`
+	Type                 string `json:"Type"`
+	Size                 string `json:"Size"`
+	Status               string `json:"Status"`
+	BackupPlanUID        string `json:"BackupPlan UID"`
+	CreationTime         string `json:"Start Time"`
+	CompletionTime       string `json:"End Time"`
+	ClusterBackupPlanUID string `json:"ClusterBackupPlan UID"`
```

Comment:
why this new field is needed? can you not use existing BackupPlanUID field? like you used backup struct for clusterBackup

remove all the code changes for it

Reply:
Now, this field is not needed . Removed this field as well as all related changes